src/js/components/container/ButtonContainer.js

```diff
@@ -29,8 +60,15 @@ class ButtonContainer extends Component {
           classes="btn btn-warning"
           text="Change Color"
           id="ColorButton"
-          onClickHandler={this.onClickHandler}
+          onClickHandler={this.handleClick}
         />
+        { this.state.displayColorPicker ? <div style={ popover } >
```

Comment:
nice use of the ternary operator!

Reply:
Thanks, it's pretty much exactly what was on the documentation for it.